jetbrains-core/src/software/aws/toolkits/jetbrains/services/lambda/execution/sam/SamInitRunner.kt

```diff
@@ -51,13 +52,28 @@ class SamInitRunner(
     }
 
     companion object {
+        private val expectedSamVersion = SemVer.parseFromText("0.6.0") ?: throw RuntimeException("SemVer parse error")
```

Comment:
Nit: put this to `message` as well

Reply:
Hesitant to put this into localized messages since this should never happen. Similar discussion here: https://github.com/aws/aws-toolkit-jetbrains/pull/414#discussion_r232465110 (give it a moment to load)

Reply:
Eh, if customer can't see it we really dont need to. But we should include the string that failed to parse